test_demo (test_login.TestLogin) ... ok
test_login_2_success (test_login.TestLogin) ... FAIL
test_register_success (test_register.TestRegister) ... ok

======================================================================
FAIL: test_login_2_success (test_login.TestLogin)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\data\jianguoyun\projects\python25\class_17_unittest\test_cases\test_login.py", line 86, in test_login_2_success
    self.assertEqual(self.data['expected'], res)
AssertionError: 'hello world' != {'code': 1006, 'copyright': 'Copyright 柠檬[82 chars]不存在'}

----------------------------------------------------------------------
Ran 3 tests in 0.060s

FAILED (failures=1)
